sympy/solvers/solveset.py

```diff
@@ -1201,24 +1200,21 @@ def _transolve(f, symbol, domain):
     How to add new class of equations
     =================================
 
-    `\_transolve` is designed in such a way that it becomes an easy
-    task to add a new class of equation solver.
-
-    The first task that needs to be done for adding your own solver
-    is to decide from where its *identification* *helper* will be
-    invoked. To do so determine the general form of the class of the
-    equation, for example the general form of the exponential equation
-    is `a*f(x) + b*g(x)` where `f(x)` and `g(x)` are power terms, so we
-    need to place the invocation condition inside `Add` case. Once the
-    place for *identification* *helper* is determined, you can add a
-    call to its *solving* *helper*.
-
-    For your class of equation you need to define your own
-    *identification* and *solving* *helpers*. The *identification*
-    *helper* should return either `True` if the given equation belongs
-    to the class otherwise `False`. *Solving* *helpers* should return
-    either the exact solution or a tractable form which `solveset`
-    can handle.
+    The design of `\_transolve` makes adding a new class of equation
+    solver easy.
+
+    Decide from where the *identification* *helper* will be invoked.
+    To do so determine the general form of the class of the equation,
+    for example the general form of the exponential equation is
+    `a*f(x) + b*g(x)` where `f(x)` and `g(x)` are power terms, so the
+    invocation condition is placed inside `Add` case.
+    Once done, add a call to its corresponding *solving* *helper*.
+
+    For the class of equation define *identification* and
```

Comment:
The previous paragraph and this one need to be edited. The last one talks about th esolving helper but this paragraph sound like *it* is trying to introduce the helpers. I think there are three steps (if there are more than Add-type transcendentals):
1. identify class
2. write identifier
3. write solvers

Reply:
Okay explaining using three steps would be better:

```
Adding a new class of equation solver is a three-step procedure:

- Identify the type of the equations

Determine the type of the class of equations to which they belong, it could be of `Add`, `Pow`, 

etc. types. Seperate internal functions are used for each type, if it is already present include

identification and solving helpers within the function otherwise add a new internal function and then 

include identification and solving helpers within it.

### code snippet can be added here

- Define the identifying helpers

- Define the solving helpers
```

Something like this is okay?